unit_test/batched/Test_Batched_TeamVectorSolveUTV2.hpp

```diff
@@ -0,0 +1,250 @@
+/// \author Kyungjoo Kim (kyukim@sandia.gov)
+
+#include "gtest/gtest.h"
+#include "Kokkos_Core.hpp"
+#include "Kokkos_Random.hpp"
+
+#include "KokkosBatched_Copy_Decl.hpp"
+#include "KokkosBatched_ApplyPivot_Decl.hpp"
+#include "KokkosBatched_Gemm_Decl.hpp"
+#include "KokkosBatched_Trsv_Decl.hpp"
+#include "KokkosBatched_UTV_Decl.hpp"
+#include "KokkosBatched_SolveUTV_Decl.hpp"
+
+#include "KokkosKernels_TestUtils.hpp"
+
+using namespace KokkosBatched;
+
+namespace Test {
+
+  template<typename DeviceType,
+           typename MatrixViewType,
+           typename VectorViewType,
+	   typename PivViewType,
```

Comment:
Minor format comment, this line is unaligned; not necessary to change in the PR, good to cleanup later.